<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoTool v1.0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>小家电自动化测试工具</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,4">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>测试模式</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0,1,0">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>选择SKU:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="skuBox"/>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>串口(COM?):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="serialBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="refreshSerial">
             <property name="text">
              <string>刷新串口</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,1">
           <item>
            <widget class="QCheckBox" name="main_funBox">
             <property name="text">
              <string>单设备主功能压测(模式/档位)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="wifi_Box">
             <property name="text">
              <string>小家电配网压测</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>118</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>测试结果</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>测试数据</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QPlainTextEdit" name="assertTextEdit">
                    <property name="placeholderText">
                     <string>输入断言数据(开机:55 11 01 00 01 01 69, 关机:55 11 01 00 01 00 68)</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>开始测试</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stopButton">
                  <property name="text">
                   <string>退出</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="resultBrowser"/>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>清除</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
